pkg/web/serviceBindingsHandler.go

```
@@ -0,0 +1,95 @@
+package web
+
+import (
+	"net/http"
+
+	"github.com/aerogear/mobile-developer-console/pkg/mobile"
+	scv1beta1 "github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/v1beta1"
+	"github.com/labstack/echo"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type BindableMobileServiceHandler struct {
+	namespace            string
+	bindableServiceCRUDL mobile.BindableMobileServiceCRUDL
+}
+
+func NewMobileServiceBindingsHandler(bindableServiceCRUDL mobile.BindableMobileServiceCRUDL, namespace string) *BindableMobileServiceHandler {
+	return &BindableMobileServiceHandler{
+		bindableServiceCRUDL: bindableServiceCRUDL,
+		namespace:            namespace,
+	}
+}
+
+func (msih *BindableMobileServiceHandler) List(c echo.Context) error {
+	mobileClientName := c.Param("name")
+	si, err := msih.bindableServiceCRUDL.List(msih.namespace, mobileClientName)
+	if err != nil {
+		c.Logger().Errorf("error listing service bindings %v", err)
+		return c.NoContent(http.StatusInternalServerError)
```

we now need to return the error code and the error message. See #102